rel/runtime_config.exs

```diff
@@ -0,0 +1,13 @@
+use Mix.Config
+
+cert = System.gen_env("DEV_CERT_PEM")
```

Comment:
did you mean?
```suggestion
cert = System.get_env("DEV_CERT_PEM")
```

Reply:
Yes I did... for the second commit I extracted it to the variable and must have mistyped... should copy paste it next time 🤦‍♂

Reply:
I was hoping you hadn't spotted that before me but you were too damn quick :P